addons/medical/functions/fnc_handleDamage_advanced.sqf

```diff
@@ -23,12 +20,23 @@
 
 #include "script_component.hpp"
 
-params ["_unit", "_selectionName", "_amountOfDamage", "_sourceOfDamage", "_typeOfProjectile", "_hitPointNumber", "_newDamage"];
+params ["_unit", "_selectionName", "_amountOfDamage", "_sourceOfDamage", "_typeOfProjectile", "_hitPointNumber", "", "_newDamage"];
 
-//Temp fix for 1.63 handleDamage changes
-if (_newDamage isEqualType objNull) then {
-    _newDamage = _this select 7;
-};
+// For burning damage we will get a ton of very small hits of damage; they are too small to create any wounds
+// Save them up in a variable and run when it is over a noticable amount
+
+if ((_typeOfProjectile == "") && {_newDamage < 0.15} && {
```

Comment:
This only kicks in for damage < 0.15 and I think most physx damage should be more than that, but can't guarentee.

The core problem is that we are increasing `bodyPartStatus` without adding new wounds.
Without wounds, the damage can't be healed, because bandageLocal does:
`if (count _openWounds == 0) exitWith {false}; // nothing to do here!`

I could try to do the fix there; so that `GVAR(healHitPointAfterAdvBandage)` or basic medical can still heal the hitpoints even if there are no wounds?